code/controllers/subsystem/SSticker.dm

```diff
@@ -768,3 +770,77 @@ SUBSYSTEM_DEF(ticker)
 	QDEL_LIST_ASSOC_VAL(load_queries)
 	records.Cut()
 	flagged_antag_rollers.Cut()
+
+/// This proc is for recording biohazard events, and blackboxing if they lived, died, or ended the round. This currently applies to: Terror spiders, Xenomorphs, and Blob.
+/datum/controller/subsystem/ticker/proc/event_blackbox(outcome = CREW_TRANSFER)
+	for(var/I in SSevents.biohazards_this_round)
+		switch(I)
+			if(GREEN_SPIDER, PRINCE_SPIDER, WHITE_SPIDER, PRINCESS_SPIDER, QUEEN_SPIDER)
+				var/output = "unknown spider type"
+				switch(I)
+					if(GREEN_SPIDER)
+						output = "Green Terrors"
+					if(PRINCE_SPIDER)
+						output = "Prince Terror"
+					if(WHITE_SPIDER)
+						output = "White Terrors"
+					if(PRINCESS_SPIDER)
+						output = "Princess Terrors"
+					if(QUEEN_SPIDER)
+						output = "Queen Terrors"
+				var/spiders = 0
+				for(var/mob/living/simple_animal/hostile/poison/terror_spider/S in GLOB.ts_spiderlist)
+					if(S.ckey)
+						spiders++
+				if(spiders >= 5 || (output == "Prince Terror" && spiders == 1)) //If a prince lives, record as win.
+					switch(outcome)
+						if(NUCLEAR)
+							SSblackbox.record_feedback("tally", "Biohazard nuclear victories", 1, output)
+						if(CREW_TRANSFER)
+							SSblackbox.record_feedback("tally", "Biohazard survives to nuclear round end", 1, output)
```

Comment:
This isn't always a nuclear end though, right? Might be worth clarifying

Reply:
Fixed